### PR TITLE
CF-662 UAE  for nova and identity

### PR DESCRIPTION
**Pull request by ChandraAddala**

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@shintasmith @usnavi Pls review.

File: message_samples/novaaccess/xml/nova-user-acess-event-v1-no-region-no-dc.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (usnavi) [inline message_samples/novaaccess/xml/nova-user-acess-event-v1-no-region-no-dc.xml]: For these types of samples, we should also have a corresponding response xml

**Comment by shintasmith** [comment]
lgtm

**Comment by shintasmith** [comment]
btw, for the future, I don't think we need to create these many xml files for each single *_access feed. Just like we don't create xml files for all the possible value of an attribute with some enumeration values. :-).

File: message_samples/novaaccess/xml/nova-user-acess-event-v1-no-region-no-dc.xml (hunk 1 above)
Comment (ChandraAddala) [inline message_samples/novaaccess/xml/nova-user-acess-event-v1-no-region-no-dc.xml]: @usnavi I had added those but pls dont merge this PR, till I fix the issue we are facing with regression tests.

**Comment by usnavi** [comment]
lgtm - @ChandraAddala - pls merge once the test deployment is successfully tested.

**Comment by ChandraAddala** [comment]
ok to test

**Comment by ChandraAddala** [comment]
@usnavi @shintasmith Can you guys take one last look and merge?

**Comment by shintasmith** [comment]
looks good

**Pull request merged**